python/src/equistore/operations/subtract.py

```diff
@@ -0,0 +1,42 @@
+from typing import Union
+
+from ..tensor import TensorMap
+from ._utils import _check_maps
+from .add import add
+
+
+def subtract(A: TensorMap, B: Union[float, TensorMap]) -> TensorMap:
+    """Return a new :class:`TensorMap` with the values being the subtract
+    of ``A`` and ``B``.
+
+    If ``B`` is a :py:class:`TensorMap` it has to have the same metadata as ``A``.
+
+    If gradients are present in ``A`` a subtract is only performed if ``B`` is
+    a :py:class:`TensorMap` as well.
+
+    :param A: First :py:class:`TensorMap` for the subtraction.
+    :param B: Second instance for the subtraction. Parameter can be a scalar or a
```

Comment:
Maybe add here the same math explanation as for `add`. It is trivial but for consistency we should do it.